core/src/main/java/org/javarosa/core/model/instance/TreeReference.java

```diff
@@ -452,8 +452,14 @@ public TreeReference contextualize(TreeReference contextRef) {
             if (contextRef.getName(i).equals(newRef.getName(i))) {
                 // Only copy over multiplicity info if it won't overwrite any
                 // existing preds or filters
+
+                // don't copy multiplicity from context when new ref's
+                // multiplicity is already bound or when the context's
+                // multiplicity is not a position (but rather an attr or
+                // template)
                 if (newRef.getPredicate(i) == null &&
-                        newRef.getMultiplicity(i) == INDEX_UNBOUND) {
+                        newRef.getMultiplicity(i) == INDEX_UNBOUND &&
+                        contextRef.getMultiplicity(i) >= 0) {
```

Comment:
@ctsims not copying over multiplicities below 0 from the context ref seems like the right call here. The call to `anchor` up above should grab those negative multiplicities, so attrs should still be setup correctly.